cmd/repo_remove.go

```diff
@@ -24,9 +24,10 @@ import (
 
 func init() {
 	repoRemoveCmd := &cobra.Command{
-		Use:   "remove",
-		Short: "Remove a artifact",
-		Long:  `Remove a artifact`,
+		Use:   "remove [artifact...]",
+		Short: "Remove an artifact",
+		Long:  `Remove an artifact`,
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
I'm not sure I want to force a minimum number of args...

Reply:
Does it make sense to run this command without any argument?